plugins/mygene/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@jbrowse/plugin-mygene",
```

Comment:
Maybe this could be a BioThings plugin and eventually also include e.g. a MyVariant adapter.

Reply:
I like that idea...always was awkward calling it mygene or myvariant

Reply:
renamed to biothings plugin